_docs/developer/getting_started/vm_install_using_vagrant.md

```diff
@@ -46,212 +52,475 @@ If you using an Intel-based Mac, you will follow the instructions below._
+   > VirtualBox and using them may not work nor are they supported. We recommend
+   > that you either download the necessary binaries from their respective steps
+   > or follow the steps outlined below for each:
+   >
+   >    VirtualBox: <https://www.virtualbox.org/wiki/Linux_Downloads>
+   >    
+   >    Vagrant: <https://developer.hashicorp.com/vagrant/downloads> 
+   >    (if that doesn't work, try: <https://vagrant-deb.linestarve.com/>)
+   
+   <br>
+   
+5. Clone [the Submitty repository](https://github.com/Submitty/Submitty) to a
+   location on your computer (the "host").
 
-   6. Click **UEFI Firmware Settings** and restart as suggested.
+   ```
+   git clone https://github.com/Submitty/Submitty.git
+   ```
+   
+   - **NOTE:** If you are not currently part of the Submitty organization on Github, you may want to
+   [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+   the repo and use the git url from your fork instead, especially if you are looking
+   to contribute.
+
+   - _OPTIONAL: If you will be developing code in one of the [companion
+   Submitty repositories](https://github.com/Submitty) (e.g., AnalysisTools, Lichen,
+   RainbowGrades, Tutorial), also clone those repositories to the same directory._
+
+      Example Organization:
+        ```
+        home
+        └── myusername
+            └── Submitty
+                │        GIT CHECKOUT
+                ├── Submitty       (main repository)
+                ├── AnalysisTools  (optional)
+                ├── Lichen         (optional)
+                ├── RainbowGrades  (optional)
+                └── Tutorial       (optional)
+        ```
+
+      _This host directory structure will be shared / synced between
+      your host operating system and the Submitty virtual machine._
+
+   <br>
+
+6. <a name="vagrant_up_instructions"></a> Navigate into the Submitty
+repository on your computer. If on Windows, you should run CMD or
+Powershell on administrator mode.
+
+   <br>
+
+7. Decide whether your development work will *require sample
+assignment submissions or autograding results.* If you do not need neither, then type ``NO_SUBMISSIONS=1`` but do **not press enter
+just yet** if on MacOS or Linux. The current command will be
+extended in the next step.
+
+   However, _if you are on Windows_, you actually **must** press
+   enter on``NO_SUBMISSIONS=1`` before proceeding, rather than
+   waiting to extend the command as MacOS and Linux users will do.
+
+   **Again, only enter ``NO_SUBMISSIONS=1`` if you do not need
+   autograding.**
+
+   Skipping the creation of the sample submissions and their
+   autograding will decrease the time to complete installation,
+   so this tradeoff may be useful to some.
+
+   <br>
+   
+   >##### Turn on Autograding after Disabling
+   >If you decide you do want autograding later on, simply type
+   >
+   >  ```
+   >  SET NO_SUBMISSIONS=
```

Comment:
These steps seem to be duplicate from the ones above,  (`SET` isn't a valid cmd on linux/mac also and you probably don't want `set` since it'll create a bash cmd argument not a variable I think)